src/fields/optional/fieldNoUiSlider.vue

```diff
@@ -1,116 +1,117 @@
 <template>
-	<div class="slider" :disabled="disabled" :class="{ 'contain-pips': containPips, 'contain-tooltip': containTooltip }"></div>  
+	<div class="slider" :disabled="disabled" :class="{ 'contain-pips': containPips, 'contain-tooltip': containTooltip }"></div>
```

Comment:
This not getting `v-attributes`?

Reply:
I didn't think about it. Since fields will be separated from the main project (in the v3), I'm not touching them too much for now.

Reply:
OK